### ISSUE TITLE
Elite request: creation of elite/13/LOS/Beckham

### ISSUE THREAD

**Issue by Wossgebber** [nikgapps/config#7634]
Hi, my name is Esteban and I am a veteran user of Custom ROMs for my Moto Z3 play (Beckham).
I would like to request automatic creation of recurring "elite" builds using a custom configuration for personal use.

I have been linking personal builds with a custom config file for several months now, in order to save installation time (since adb sideloading multiple packages to a slow, lineage-based recovery can be... onerous).

Many thanks to Nikhil and his team for the work on this project, and now for inviting me to test automatic package building!

Additionally, I made the corresponding donation via paypal.

**Comment by nikgapps** [comment]
Hi @Wossgebber thank you for the donation, can you specify the name of the folder you'd like your elite builds to be part of on Sourceforge.net?

I will provide you access to that folder and you can then create one last pull request under elite/<android_version>/<the_folder_you_specify>

You can upload any number of config files in that folder and the builds will be created shortly after the pull request is merged and also with every new release.

Let me know if you have questions.

**Comment by Wossgebber** [comment]
Sure: elite/13/LOS/Beckham

**Comment by nikgapps** [comment]
@Wossgebber let me rephrase.

The format of the config path is

elite/<android_version>/<the_folder_you_specify>

Which doesn't fit in path you provided. elite/13/LOS/Beckham 

I can provide you access to a folder named Beckham and you can send a pull request by uploading your config file to elite/13/Beckham/<yourconfig.config>

**Comment by nikgapps** [comment]
Now, you have access to Beckham folder. Feel free to send your pull request.

**Comment by Wossgebber** [comment]
I think I understand, it could be elite/13/Wossgebber/LOS/Beckham?

**Comment by nikgapps** [comment]
@Wossgebber the path has to be elite/13/<folder_name> for following reasons

1. It allows us to use the same folder name on Sourceforge.net and upload the builds to your dedicated folder
2. 2. It keeps the path streamlined for every Elite user
3. 3. Subdirectoriea are hard to maintain, the system will get confused if you upload a file in elite/13/Wossgebber/Beckham/some_config.config while LOS still exists.

What I can offer you is elite/13/Beckham folder or elite/13/Wossgebber folder (or both if you need)

Perhaps you can tell me what you are trying to achieve with elite/13/Wossgebber/Beckham/LOS and I can help you with that.

If you're trying to get a build for Los, you can name your config in such way (Beckham-LOS.config)

I've already provided access to you to elite/13/Beckham and you can upload any number of config files in that folder. Let me know your thoughts.

**Comment by nikgapps** [comment]
Closing as the setup is complete

**Comment by nikhilmenghani** [comment]
@Wossgebber which android version nikgapps elites build are you using? android 13 or 14?

**Comment by nikhilmenghani** [comment]
@Wossgebber in preparation of android 15, I am deleting elite config for android 13, let me know if you still need it.
